prime-router/src/testIntegration/kotlin/datatests/mappinginventory/xtn/XTNTests.kt

```diff
@@ -0,0 +1,86 @@
+package gov.cdc.prime.router.datatests.mappinginventory.xtn
+
+import gov.cdc.prime.router.Report
+import gov.cdc.prime.router.cli.tests.CompareData
+import gov.cdc.prime.router.datatests.TranslationTests
+import org.junit.jupiter.api.Test
+
+class XTNTests {
+    fun shouldTranslateLosslessly(inputFile: String, outputFile: String): CompareData.Result {
```

Comment:
I like this approach of reusing functions from TranslationTests for "unit" tests, though I think we can improve the setup. Feel free to save this for future stories.

I'm not a big fan of the function name `shouldTranslateLosslessly`; it really has nothing to do with losslessness, it just compares a result to an expected output. Maybe `tranlsateAndCompareFile` instead? And since we could easily imagine the same test being used elsewhere, we should probably pull this function out of `XTNTests` and into a more general location.

As for parameters, instead of `outputFile`, maybe we call it `expectedOutputFile`? The same issue probably already exists everywhere in TranslationTests, so not too important. We could also include parameters for input/output formats (or automatically determine those formats based on the file names), but that can probably wait until we have tests that would be using different ones.